CoreFoundation/Parsing.subproj/CFXMLInterface.c

```diff
@@ -1179,6 +1236,131 @@ void _CFXMLDTDNodeSetPublicID(_CFXMLDTDNodePtr node, const unsigned char* public
     }
 }
 
+// Namespaces
+_CFXMLNodePtr _Nonnull * _Nullable _CFXMLNamespaces(_CFXMLNodePtr node, CFIndex* count) {
```

Comment:
we might wanna use a CFArrayRef here because this requires freeing the return value